test/jdk/tools/launcher/RunpathTest.java

```diff
@@ -24,7 +24,7 @@
 /*
  * @test
  * @bug 7190813 8022719
- * @summary Check for extended  RPATHs on *nixes
+ * @summary Check for extended RPATHs on Linux
```

Comment:
Pre-existing but really the restriction to Linux should be via `@requires` not a runtime test.

Reply:
Fixed